aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md

```diff
@@ -150,27 +150,27 @@ Once you have completed the steps in the preceding sections, you should test you
 4. At this point the ASP.NET Identity tables are created on the MySQL Database, and the user is registered and logged into the application:
 
    [Click the following image to expand it. ]
-    [![](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image6.jpg)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image5.jpg)
+    [![Screenshot of A S P dot NET website after user has completed registration. Tab with Hello greeting, followed by username, is highlighted in menu at upper right.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image6.jpg)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image5.jpg)
 
 ### Installing MySQL Workbench tool to verify the data
 
 1. Install the **MySQL Workbench** tool from the [MySQL downloads page](http://dev.mysql.com/downloads/windows/installer/)
 2. In the installation wizard: **Feature Selection** tab, select **MySQL Workbench** under **applications** section.
-3. Launch the app and add a new connection using the connection string data from the Azure MySQL database you created at the begging of this tutorial.
+3. Launch the app and add a new connection using the connection string data from the Azure MySQL database you created at the beginning of this tutorial.
 4. After establishing the connection, inspect the **ASP.NET Identity** tables created on the **IdentityMySQLDatabase.**
 5. You will see that all ASP.NET Identity required tables are created as shown in the image below:
 
    [Click the following image to expand it. ]
-    [![](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image8.jpg)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image7.jpg)
+    [![Screenshot of My S Q L Workbench tool dialog. A S P dot NET Identity tables created on the Identity My S Q L Database are highlighted at lower left.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image8.jpg)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image7.jpg)
 6. Inspect the **aspnetusers** table for instance to check for the entries as you register new users.
 
    [Click the following image to expand it. ]
-    [![](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image26.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image25.png)
+    [![Screenshot of a s p net users table, with entries displaying Id, UserName, PasswordHash, SecurityStamp, and Discriminator columns.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image26.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image25.png)
```

Comment:
Screenshot of A S P dot net users table, with entries displayed I D, User Name, Password Hash, Security Stamp, and Discriminator columns (may need to be adjusted for character limit)